### ISSUE TITLE
Package 'Newtonsoft.Json' 9.0.1 has a known moderate severity (coverlet.core)

### ISSUE THREAD

**Issue by Bertk** [coverlet-coverage/coverlet#1505]
Project coverlet.core uses [vulnerable Package 'Newtonsoft.Json 9.0.1](https://github.com/advisories/GHSA-5crp-9r3c-p9vr)'. This assembly is also added coverlet Nuget packages.

<img width="624" alt="image" src="https://github.com/coverlet-coverage/coverlet/assets/2315215/428ac34e-36fb-47cd-a9fb-7d375d22dcdd">

The version is defined in [Directory.Build.targets](https://github.com/coverlet-coverage/coverlet/blob/master/Directory.Build.targets) with comment "Do not upgrade".
```xml
    <!-- Do not upgrade this version or we won't support old SDK -->
    <PackageReference Update="Newtonsoft.Json" Version="9.0.1" />
    <PackageReference Update="NuGet.Packaging" Version="5.4.0" />
    <PackageReference Update="ReportGenerator.Core" Version="4.6.0" />
```

When will the **Newtonsoft.Json** package version be upgraded to **V13.0.3**?

**Comment by SeijiSuenaga** [comment]
@Bertk do you know when a version of Coverlet with this security patch might be published?  I see the PR was merged 5 months ago, but the latest published version 6.0.0 was 9 months ago.  Sorry for the ping if the plan is already documented somewhere, but I couldn't find it.

**Comment by Bertk** [comment]
coverlet nuget packages will be released soon. By,the way the Newtonsoft.Json package will be replaced (https://github.com/coverlet-coverage/coverlet/pull/1513).
